src/app/components/forms/input/double-input/terra-double-input.component.spec.ts

```diff
@@ -58,4 +66,61 @@ describe('TerraDoubleInputComponent', () =>
     {
         expect(component).toBeTruthy();
     });
+
+    it('should be invalid', () =>
+    {
+        component.value = 'Test';
+        expect(component.value).not.toEqual(jasmine.any(Number));
+    });
+
+    it('should be valid', () =>
+    {
+        const formControl:FormControl = new FormControl(testValue, [Validators.pattern(TerraRegex.DOUBLE)]);
+
+        component.value = testValue;
+        component.validate(formControl);
+
+        expect(component.isValid).toBeTruthy();
+    });
+
+    it('should be true if greater than 0', () =>
+    {
+        component.inputDecimalCount = 1;
+        expect(component.inputDecimalCount).toBeGreaterThan(0);
+    });
+
+    it('should be false as default', () =>
+    {
+        expect(component.inputIsPriceInput).toBe(false);
+    });
+
+    it('should be price input if true is set', () =>
+    {
+        component.inputIsPriceInput = true;
+        expect(component.inputIsPriceInput).toBe(true);
```

Comment:
Leider auch kein richtiger Test.

Reply:
![image](https://user-images.githubusercontent.com/9987324/48466090-145d8380-e7e5-11e8-827c-cefdf47e117c.png)

Reply:
dann sind die tests in der infobox component auch nicht richtig